Localization/app.json

```
@@ -438,6 +438,10 @@
                 "placeholder": {
                     "label": "Label",
                     "content": "Content"
+                },
+                "verified": {
+                    "short": "Verified at %s",
```

Slight grammar tweak (since it includes the date):

```suggestion
                    "short": "Verified on %s",
```